### PR TITLE
[RFC 0044] Disband the Nix Core Team

### PR DESCRIPTION
**Pull request by shlevy**
[Rendered](https://github.com/shlevy/rfcs/blob/disband-nix-core/rfcs/0044-disband-nix-core.md)

### CODE REVIEW

File: rfcs/0044-disband-nix-core.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* There is not perfect overlap between the RFC process coverage and
+  what the core team was intended to cover.
+
+# Alternatives
+[alternatives]: #alternatives
+
+* Keep the core team around in its current form and responsibilities.
+  Would require a fresh attempt to follow through on the relevant
+  committments to be practical.
+* Reform the core team based on what we've learned, including possibly
```
Comment (grahamc) [inline rfcs/0044-disband-nix-core.md]: I would love to hear what was learned anyway!

**Comment by edolstra** [comment]
I would like to propose the members of the core team as shepherds: @edolstra @shlevy @peti @vcunat @copumpkin

File: rfcs/0044-disband-nix-core.md (hunk 1 above)
Comment (vcunat) [reply rfcs/0044-disband-nix-core.md]: I learned I was too optimistic (about my time/energy) and thus ended up contributing through this team even less than I feared when it was being formed.  That didn't result into me really "blocking" anything, I hope.

**Comment by shlevy** [comment]
Shepherd team is @edolstra @peti @vcunat @copumpkin, with @edolstra as leader. I'd propose we're ready for FCP.

File: rfcs/0025-nix-core-team.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 co-authors: Daniel Peebles, Eelco Dolstra, Peter Simons, Shea Levy, Vladimír Čunát
-related-issues:
+related-issues: #44 (disbands the team)
 ---
 
+# Superceded!
+
+This RFC is superceded by [RFC 44](./0044-disband-nix-core.md). The original text
+is preserved below.
+
```
Comment (vcunat) [inline rfcs/0025-nix-core-team.md]: Nitpick: that's a repeated [typo](https://en.wiktionary.org/wiki/supercede#Verb) ;-)

Comment (shlevy) [reply rfcs/0025-nix-core-team.md]: Fixed, thanks!

**Comment by edolstra** [comment]
This RFC is now in Final Comment Period.

**Comment by globin** [comment]
This RFC has been accepted!

**Pull request merged**